spec/models/reference_spec.rb

```diff
@@ -1,5 +1,7 @@
 require 'rails_helper'
 
-RSpec.describe Reference, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+describe Reference do
+  it 'is valid with name address position' do
+    expect(build(:reference)).to be_valid
+  end
```

Comment:
Could you do some invalid tests here as well? Like you did with `MontessoriTrainingSpec`?